client/rpc/dex_client.go

```diff
@@ -5,12 +5,17 @@ import (
 	"fmt"
 	"github.com/binance-chain/go-sdk/common/types"
 	"github.com/binance-chain/go-sdk/types/tx"
+
 )
 
 const (
 	AccountStoreName = "acc"
```

Comment:
could you do `gofmt -w -s xx.go` to these modified files.

Reply:
solved!